staging/src/k8s.io/kubectl/pkg/cmd/set/set_image.go

```diff
@@ -66,7 +66,7 @@ type SetImageOptions struct {
 
 var (
 	imageResources = i18n.T(`
-  	pod (po), replicationcontroller (rc), deployment (deploy), daemonset (ds), replicaset (rs)`)
+  	pod (po), replicationcontroller (rc), deployment (deploy), daemonset (ds), cronjob (cj), replicaset (rs)`)
```

Comment:
Similar to my other comment.  Would `statefulset` be valid here as well?

Reply:
As I replied to the other comment, I updated to include `statefulset`.